content/influxdb/v2.0/tools/repl.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Interactive Flux REPL
+seotitle: Interactive Flux REPL
+description: Use the Flux REPL to test queries and interact with data.
+influxdb/v2.0/tags: [flux]
+menu:
+  influxdb_2_0:
+    parent: Tools
+weight: 104
+---
+
+Use the Flux REPL (Read–Eval–Print Loop) to execute Flux scripts and interact with InfluxDB and other data sources.
+[Build the REPL](#build-the-repl) from the Flux source code.
+
+## Build the REPL
+
+See the [Flux repository README](https://github.com/influxdata/flux#requirements) for instructions on building the REPL.
+
+## Use the REPL
+
+Once built, open a new REPL session with:
+
+```sh
+./flux repl
+```
+
+To query data from InfluxDB, provide the organization, token, and host parameters to the [`from()` function](/v2.0/reference/flux/stdlib/built-in/inputs/from/).
+
+<!-- Variables in the REPL are immutable in the current session. -->
+
```

Comment:
Probably want to add a section about multi-line versus single-line entries. There's no way to enter a newline character directly from the keyboard because `return` executes the current line. You can however, pasted newlines in from the clipboard. So you can paste in and execute a multi-line Flux script, but you can't type one out manually inside the REPL.